ansible/configure/roles/database-import/tasks/main.yml

```diff
@@ -10,12 +10,20 @@
     dest: "/opt/rh/postgresql92/root/var/lib/pgsql/data/{{appliances[inventory_hostname].dbdump}}"
   when: is54
 
-# 5.5 appliances have ~1GiB in the /tmp directory, thus most database exports will fill the
+# 5.5+ appliances have ~1GiB in the /tmp directory, thus most database exports will fill the
 # tmp partition and we must rely on wget rather than the ansible get_url module
 - name: Get database dump (5.5.x appliances)
   shell: "wget {{db_dumps[appliances[inventory_hostname].dbdump]}} -O /var/opt/rh/rh-postgresql94/lib/pgsql/data/{{appliances[inventory_hostname].dbdump}}"
   when: is55
 
+- name: Get database dump (5.6.x appliances)
+  shell: "wget {{db_dumps[appliances[inventory_hostname].dbdump]}} -O /var/opt/rh/rh-postgresql94/lib/pgsql/data/{{appliances[inventory_hostname].dbdump}}"
+  when: is56
```

Comment:
yes, I had done that in a commit added later on https://github.com/arcolife/cfme-performance/commit/abbbdead337c329f8b77c7a532658a39baba8f3a#diff-e3c200327cea1ff0647622cadd19db25R48